test_cases.md

```diff
@@ -0,0 +1,107 @@
+# List of testing cases before release
+
+### Accuracy check via simulation: IBIS workflow
+
+
+**Description:**
+
+We are checking accuracy of IBIS workflow with `simulate` command. 
+It will take roughly an hour. 
+
+**Command:**
+
+```
+docker run --rm --privileged -it -v /media/data1/relatives:/media -v /etc/localtime:/etc/localtime:ro genx_relatives:latest launcher.py simulate --ref-directory /media/big_ref --cores 8 --directory /media/runs/sim-ibis-small --flow ibis --assembly hg37 --real-run
+```
+
+
+**Desired result:**
+
+Accuracy of almost 100\% for the first 3 degrees and better than 0\% accuracy for 4-6.
```

Comment:
By accuracy you mean recall? "better than 0\%", it's almost 100 for 4-6, and better then 0 for 7-9.